modules/ezcontent_node/modules/ezcontent_smart_article/ezcontent_smart_article.module

```diff
@@ -73,6 +76,42 @@ function ezcontent_smart_article_form_node_form_alter(&$form, FormStateInterface
     }
 
   }
+
+  // Smarttags process.
+  $bundle = '';
+  $entityType = '';
+  $form_object = $form_state->getFormObject();
+  if ($form_object instanceof ContentEntityForm) {
+    $bundle = $form_object->getEntity()->bundle();
+    $entityType = $form_object->getEntity()->getEntityTypeId();
+  }
+  $form['#attached']['library'][] = 'ezcontent_smart_article/ezcontent_smart_article_libs';
```

Comment:
We should attach the library, only when the condition is met. Place it after the condition `if (!empty($entityType) && !empty($bundle)) {`

Reply:
The entire code for `Smart Tags` was lying outside the form_id condition. Moved it inside the condition where the library was already getting attached.